Dockerfile.centos7:xdmod-base

```diff
@@ -0,0 +1,58 @@
+FROM centos:7.9.2009
```

Comment:
You mentioned that this was copied from the xdmod project, but I'm not sure that we need it. Installing the `xdmod` rpm itself will bring in any necessary dependencies, so the other Dockerfiles should be fine if they were simply based on a recent CentOS image instead.

Reply:
At this stage the xdmod-base loads tooling that is convenient (vi, wget ... ) that is use in building xdmod.

Reply:
We're not actually building xdmod, right? We're just consuming the RPM. But if it's convenient to get things going might as well keep it for now; we can also revisit later.

Reply:
The base RPM doesn't include all of its dependencies.